common/build.gradle

```diff
@@ -1,10 +1,13 @@
-
-
 plugins {
     id "org.sonarqube" version "2.5"
 }
 
 apply plugin: 'com.android.library'
+apply plugin: 'pmd'
```

Comment:
Making a personal note here: we need to run `pmd` and ensure it's clean prior to publish. Will make an issue for this and link here.

Reply:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/issues/82